scripts/gui/microbe_editor.mjs

```diff
@@ -126,7 +130,10 @@ export function setupMicrobeEditor(){
 
         element.element.addEventListener("click", (event) => {
             event.stopPropagation();
-            onSelectNewOrganelle(element.organelle);
+
+            if(!$(element.element).hasClass("DisabledButton")) {
```

Comment:
I think using jquery here is unnecessary as the modern dom classlist object should have a contains method.

Reply:
Fixed